sdk/documentintelligence/Azure.AI.DocumentIntelligence/samples/Sample_GetAndListOperations.md

```diff
@@ -0,0 +1,91 @@
+# Get and List Document Model Operations
+
+This sample demonstrates how to get and list operations of a Document Intelligence resource. Note that operation information only persists for 24 hours.
+
+To get started you'll need a Cognitive Services resource or a Document Intelligence resource. See [README][README] for prerequisites and instructions.
+
+## Creating a `DocumentIntelligenceAdministrationClient`
+
+To create a new `DocumentIntelligenceAdministrationClient` you need the endpoint and credentials from your resource. In the sample below you'll use a Document Intelligence API key credential by creating an `AzureKeyCredential` object that, if needed, will allow you to update the API key without creating a new client.
+
+You can set `endpoint` and `apiKey` based on an environment variable, a configuration setting, or any way that works for your application.
+
+```C# Snippet:CreateDocumentIntelligenceAdministrationClient
+string endpoint = "<endpoint>";
+string apiKey = "<apiKey>";
+var client = new DocumentIntelligenceAdministrationClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
+```
+
+## Get and List Document Model Operations
+
+The method `GetOperations` returns a list of `OperationDetails` instances. The instances returned by this method contain general information about the operation, such as its ID, its status, and the `PercentCompleted` property to track progress, but it does not include details about the result or errors that happened during its execution.
```

Comment:
Here you describe the general `GetOperations` method as well as the `GetOperation` method which is more specific and returns extra properties. 

nit: Similar to my other comment, the sample uses `GetOperationsAsync` and `GetOperationAsync` instead.

nit: The code snippet showcases both methods in reverse order, i.e., `GetOperationAsync` is called first. To make reading it flow better, I wonder if the code snippet can be refactored so that `GetOperationsAsync` comes first instead.

Reply:
I think you have a point. The original sample was inverted, actually, but for some reason the code felt weird to me. I'll revisit this one after the release since it's low-priority right now: https://github.com/Azure/azure-sdk-for-net/issues/40061